rust-symcrypt/README.md

```diff
@@ -43,13 +43,10 @@ HKDF:
 - HmacSha384
 - HmacSha512
 
-GCM:
-- Encryption ( in place )
-- Decryption ( in place )
-
-ChaCha:
-- Encryption ( in place )
-- Decryption ( in place )
+Encryption: 
+- Gcm Encrypt/Decrypt
+- ChaCha Encrypt/Decrypt
+- Aes CBC Encrypt/Decrypt

```

Comment:
>Aes CBC [](http://example.com/codeflow?start=2&length=7)

Nit: kind of a weird mix of capitalization in this file. Previous ones seem to match the struct names (Pascal case), but this one and the ECC/RSA algorithms below are more like how you'd write them in documentation (they're all acronyms).

If the latter, this should be AES-CBC.

Reply:
Thanks for the catch, changing to AES-CBC